mPower2/mPower2/PassiveCollectors.swift

```diff
@@ -533,10 +533,21 @@ class PassiveGaitCollector : NSObject, PassiveLocationTriggeredCollector {
                 }
             }
             
-            // This will get a good-enough initial fix and set a geofence.
-            self.locationManagerPaused = true
-            self.locationManager!.desiredAccuracy = kCLLocationAccuracyBest
-            self.locationManager!.requestLocation()
+            if self.locationManagerPaused {
```

Comment:
So... you deleted the line above where you set `self.locationManagerPaused = true` but here you are checking for paused? Is that also set elsewhere?

Reply:
Yes, it's initialized as `true`, and *shouldn't* have been being set here since this can get called when the location manager is currently receiving updates. It gets set to `false` when updates are started/restarted (i.e. when you leave the geofenced area) and back to `true` when iOS pauses location updates.